src/raygun.loader.js

```diff
@@ -218,6 +229,41 @@
     }
   };
 
+  function ping() {
+    if(metadata.ping.failedPings > 10) {
+        clearInterval(metadata.ping.pingIntervalId);
+    }
+
+    var apiKey = window.Raygun.Options._raygunApiKey;
+   
+    var url = "https://api.raygun.io/ping?apiKey=" + apiKey;
+    var data = {
+        crashReportingEnabled: crashReportingEnabled || false,
+        realUserMonitoringEnabled: realUserMonitoringEnabled || false,
+        providerName: "raygun4js",
+        providerVersion: '{{VERSION}}'
+    };
+
+    fetch(url, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(data)
+    }).then(function(response) {
+    if (response.ok) {
+      metadata.ping.failedPings = 0;
+    } else {
+      // Request failed
+      metadata.ping.failedPings++;
+    }
+  })
```

Comment:
This is a bit ugly here, we normally put the .catch in the line. No worries for now